Sources/Internal/GrpcClient.swift

```diff
@@ -155,49 +140,28 @@ actor GrpcClient: CustomStringConvertible {
       connectorName: connectorName,
       request: request
     )
+    let requestString = try ": " + grpcRequest.jsonString()
```

Comment:
nit: I think it might be easier to read the log statements below if the ": " is included there instead of in `requestString`.